backend/videoService/vonageVideoService.ts

```diff
@@ -59,6 +66,22 @@ class VonageVideoService implements VideoService {
     await this.vonageVideo.stopArchive(archiveId);
     return 'Archive stopped successfully';
   }
+
+  async enableCaptions(sessionId: string): Promise<EnableCaptionResponse> {
+    const requestToken = this.generateToken(sessionId);
+    const { token } = requestToken;
+    return this.vonageVideo.enableCaptions(sessionId, token);
```

Comment:
Our sample node repo has this wrapped in a `try-catch`. Can we ensure there's proper error-handling from the vonage video service?

Reply:
I was mostly going for consistency with the start/stop archive here but definitely, having the `try-catch` has its benefit. I'm also going to add it to the opentok video service for consistency. thanks Chris!